### PR TITLE
[WEEKLY RELEASE] HotShot - rc-0.5.79 

### PR DESCRIPTION
**Pull request by dailinsubjam**
Closes #<ISSUE_NUMBER>

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -57,24 +57,24 @@ dyn-clone = "1.0"
...
-hotshot-builder-core = { git = "https://github.com/EspressoSystems/marketplace-builder-core", tag = "0.1.53" }
-marketplace-builder-core = { git = "https://github.com/EspressoSystems/marketplace-builder-core", tag = "0.1.53" }
-marketplace-builder-shared = { git = "https://github.com/EspressoSystems/marketplace-builder-core", tag = "0.1.53" }
-hotshot-events-service = { git = "https://github.com/EspressoSystems/hotshot-events-service.git", tag = "0.1.51" }
-hotshot-orchestrator = { git = "https://github.com/EspressoSystems/hotshot", tag = "0.5.78-patch4" }
-hotshot-query-service = { git = "https://github.com/EspressoSystems/hotshot-query-service", tag = "0.1.66" }
-hotshot-stake-table = { git = "https://github.com/EspressoSystems/hotshot", tag = "0.5.78-patch4" }
+hotshot-builder-api = { git = "https://github.com/EspressoSystems/HotShot.git", branch = "bump/rc-0.5.79" }
+hotshot-builder-core = { git = "https://github.com/EspressoSystems/marketplace-builder-core", branch = "hotshot/rc-0.5.79" }
+marketplace-builder-core = { git = "https://github.com/EspressoSystems/marketplace-builder-core", branch = "hotshot/rc-0.5.79" }
```
Comment (tbro) [inline Cargo.toml:64]: Looks good. Only question: should we move these from `branch` to `tag` before we merge?

Comment (dailinsubjam) [reply Cargo.toml:64]: Yes! I'll update them after weekly release PR are merged in other repos.

**Review by tbro — APPROVED** [review]
LGTM

**Comment by alxiong** [comment]
let's try to fix the Slow Test. Something is clogging it. I understand it's annoying to wait for 40min to merge the PR. 
We should try not to break the CI and investigate asap.

**Pull request merged**